### DISCUSSION TITLE
查询参数和字符串校验章节部分表述错误

### DISCUSSION THREAD

**Discussion by liufei1997** [fastapi/fastapi#8470] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
官方文档表述错误
```

### Description

![image](https://user-images.githubusercontent.com/58357252/165206822-c6ae4314-d0eb-489c-b1cd-229489b134b4.png)
图片中红色方框位置，应该使用None，而不是...

### Operating System

Linux

### FastAPI Version

官方文档

### Python Version

3.7

### Additional Context

![image](https://user-images.githubusercontent.com/58357252/165207006-3c281fd3-d5fc-4406-a836-455ebfbf0f61.png)

**Comment by yinziyan1206** [comment, marked as answer]
It is the same in original(English) edition
